tests/test_etl.py

```diff
@@ -74,6 +76,23 @@ def test_etl(raw_data):
     assert len(filter_out_rows) == 1
     assert filter_out_rows[0]["page_id"] == expected_page_id
 
+    # page_namespace is correctly marked as 0 for all rows
+    assert (
+        ddf.where(F.col("page_namespace") == 0)
```

Comment:
Tests are great.

If more cases come up where we need to check that a condition applies to all `expected_num_records`, we could generalise this behaviour into a function. Could this be a nice first task if we need to onboard new folks on this codebase?

Reply:
I ended up removing these but yes I agree it could probably be a really good onboarding task!